treeherder/seta/preseed.py

```diff
@@ -94,9 +134,14 @@ def create_new_entry(job):
         job['expiration_date'] = THE_FUTURE
 
     logger.info("Adding a new job priority to the database: %s", job)
+
+    testtype = job['testtype']
+    buildtype = job['buildtype']
```

Comment:
why is buildtype pulled out here?

Reply:
I did that during debugging but removed the debugging statements.